README.md

```diff
@@ -542,13 +541,12 @@ uart:~$ xu unpause 2
 domain:2 unpaused
 ```
 
-Destroy DomD - DomD will be destroyed and Xen domain id "2" will not be accessible any more.
-
+To destroy the rpi_5_domd, run:
 ```
 uart:~$ xu destroy 2
-(XEN) avc:  denied  { destroy } for source=d0 target=d2 scontext=system_u:system_r:dom0_t tcontext=system_u:system_r:unlabeled_t tclass=domain
 domain:2 destroyed
 ```
+After destroying the Xen domain, ID "2" will no longer be accessible.
```

Comment:
This change is quite interesting, but it doesn't comply with the style of the previous steps, so it's better not to apply it.